pkg/console/subresource/route/route.go

```diff
@@ -88,6 +101,41 @@ func Validate(route *routev1.Route) (*routev1.Route, bool) {
 	return route, changed
 }
 
+func IsCustomRouteSet(operatorConfig *operatorv1.Console) bool {
+	if len(operatorConfig.Spec.Route.Hostname) != 0 {
+		return true
+	}
+	return false
+}
+
+// Check if reference for secret holding custom serving-cert is set
+func IsCustomRouteSecretSet(operatorConfig *operatorv1.Console) bool {
+	if len(operatorConfig.Spec.Route.Secret.Name) != 0 {
+		return true
+	}
+	return false
+}
+
+func ValidateCustomRoute(operatorConfig *operatorv1.Console, client routeclient.RoutesGetter) error {
```

Comment:
I'm not sure I follow the logic entirely here. 
Its not really validating the custom route its.... checking if the custom is still on the default ingress domain?

Reply:
yeah, probably bad naming hare. Maybe `isDefaultIngressDomain` ?